src/perfcollect/perfcollect

```diff
@@ -2153,6 +2312,11 @@ ProcessArguments $@
 # Ensure prerequisites are installed.
 EnsurePrereqsInstalled
 
+if [ "$useEventPipe" == "1" ]  && [ "$1" != "stop" ]
+then
+    EnsureDotNetTraceToolIsInstalled
+fi
+
```

Comment:
Can you please add a regression test that uses `dotnet-trace`?

Reply:
Ok, I missed the test folder. I will add a regression test.

Reply:
I tried to add a regression test (it allowed me to find stuff I forgot to add in the script: installing curl). But, to run the script with -dotnet-trace, we need an .NET app running in the container. I'm not sure it's possible to do that in the current state. Do you want me to add this?

Reply:
Yes, that would be great.